### PR TITLE
Increases spooky factor 100%

### PR DESCRIPTION
**Pull request by ToGWtF**
Refer to issue #1124
### Intent of your Pull Request

todo : changelog  ,  make them slow you down   ,  make them partially blind you

### CODE REVIEW

**Comment by ParagonPrime** [comment]
**_BEDSHEET TOGAS WHEN??????????????????????????_**

**Comment by FluffySurvivor** [comment]
> make them slow you down

no

**Comment by ToGWtF** [comment]
@FluffySurvivor going to be a tradeoff.  If it covers your face to make you unknown without an ID on/cover your clothes, its getting slowdown

**Comment by FluffySurvivor** [comment]
@ToGWtF 
give the gasmask a slowdown too while you're at it kay?

File: code/game/objects/structures/bedsheet_bin.dm [hunk 1]
```diff
@@ -61,6 +68,7 @@ LINEN BINS
 	desc = "You've never felt more free than when sleeping on this."
 	icon_state = "sheetUSA"
 	item_color = "sheetUSA"
+	slot_flags = SLOT_BACK
```
Comment (Super3222) [inline code/game/objects/structures/bedsheet_bin.dm:71]: Wait what was the point of adding this to each one when the original `/obj/item/weapon/bedsheet` already has SLOT_BACK.

Comment (ToGWtF) [inline code/game/objects/structures/bedsheet_bin.dm:71]: because they dont have slot_head sprites

**Comment by Time-Green** [comment]
Why is this not merged yet?

**Comment by oisin100** [comment]
> Why is this not merged yet?

Not finished yet.

**Comment by FluffySurvivor** [comment]
no slowdown or riot

**Comment by Super3222** [comment]
If someone with clumsiness (or instead a lotta brain damage) bumps into a wall with a bedsheet over them, they should bash into it.

**Comment by ParagonPrime** [comment]
inb4

straightjacket, bedsheet, and muzzle the clown and force him to run around labour camp forever

i'm finding so many fun new torture ideas as hos

**Comment by oisin100** [comment]
You should recreate this PR on the new repository @ToGWtF

**Pull request closed without merging**